mobile/src/navigation/course-module/course-module.navigation.tsx

```diff
@@ -30,10 +35,14 @@ const CourseModule: FC = () => {
 
   return (
     <Tab.Navigator screenOptions={SCREEN_OPTIONS}>
```

Comment:
As far as I remember we have other tabs with conditional render (on courses screen, maybe somewhere else), add this logic for them too